pkg/syncer/storagepool/intended_state.go

```diff
@@ -189,36 +185,44 @@ func newIntendedState(ctx context.Context, ds *cnsvsphere.DatastoreInfo,
 	}, nil
 }
 
-func isRemoteVsan(ctx context.Context, dsprops *dsProps, clusterID string, vcclient *vim25.Client) (bool, error) {
-	log := logger.GetLogger(ctx)
+// A remote Datastore is a vSAN datastore that belongs to a different VC cluster and is mounted on this VC.
```

Comment:
Can you change method comment.
It seems here we are talking about mounted vsan cluster from another vCenter which is not the case here.

Reply:
Reverted it back as we're already defining it in the `intendedState`.